java-connectors/kafka-connect-azure-eventhubs/src/main/java/io/lenses/java/streamreactor/connect/azure/eventhubs/source/AzureEventHubsSourceTask.java

```diff
@@ -48,10 +48,10 @@ public void start(Map<String, String> props) {
     AzureEventHubsConfig azureEventHubsConfig = new AzureEventHubsConfig(props);
     TopicPartitionOffsetProvider topicPartitionOffsetProvider = new TopicPartitionOffsetProvider(offsetStorageReader);
 
-    ArrayBlockingQueue<ConsumerRecords<String, String>> recordsQueue = new ArrayBlockingQueue<>(
+    ArrayBlockingQueue<ConsumerRecords<byte[], byte[]>> recordsQueue = new ArrayBlockingQueue<>(
         RECORDS_QUEUE_DEFAULT_SIZE);
     blockingQueueProducerProvider = new BlockingQueueProducerProvider(topicPartitionOffsetProvider);
-    BlockingQueuedKafkaProducer producer = blockingQueueProducerProvider.createProducer(
+    ByteBlockingQueuedKafkaProducer producer = blockingQueueProducerProvider.createProducer(
```

Comment:
Does the BlockingQueue need to have `byte`? 
Also the Kafka in the name would suggest that somehow there's a Kafka producer which uses a blocking queue. And in reality this class is a writer to the blocking queue, it does not produce record to Kafka.

Reply:
Yeah, the ordering of words was kind of unfortunate, so I changed it to `KafkaByteBlockingQueuedProducer` which should better reflect where it produces to. I left `Kafka` and `Byte` part because `BlockingQueueProducer` interface is oblivious to type and where it gets data from :)